File: PersonnelManagement/PersonnelManagement/Models/Vacationtype.cs
using System;
using System.Collections.Generic;

namespace PersonnelManagement.Models
{
    public partial class Vacationtype
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Name2 { get; set; }
        public sbyte Social { get; set; }
        public sbyte Cadet { get; set; }
        public sbyte Military { get; set; }
        public sbyte Civil { get; set; }
        public sbyte Transferworkyear { get; set; }
        public sbyte Main { get; set; }
        public sbyte Maternity { get; set; }
        public int Durationmax { get; set; }
        public int Order { get; set; }
    }
}

File: PersonnelManagement/PersonnelManagement/Front/classes/vacationtype.ts
export default class Vacationtype {
    id: number;
    name: string;
    name2: string;
    social: boolean;
    cadet: boolean;
    civil: boolean;
    military: boolean;
    transferworkyear: boolean;
    main: number;
    maternity: number; // декретный отпуск
    durationmax: number; // максимальная длительность отпуска. 0 если не установлен
    order: number;
}